packages/catalog-model/src/kinds/GroupEntityV1alpha1.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as yup from 'yup';
+import type { Entity } from '../entity/Entity';
+import type { EntityPolicy } from '../types';
+
+const API_VERSION = ['backstage.io/v1alpha1', 'backstage.io/v1beta1'] as const;
+const KIND = 'Group' as const;
+
+export interface GroupEntityV1alpha1 extends Entity {
+  apiVersion: typeof API_VERSION[number];
+  kind: typeof KIND;
+  spec: {
+    type: string;
+    owner: string;
```

Comment:
Should we really model this? @Rugvip @benjdlambert @soapraj etc, we don't even use this internally right? If we are to model key people of a group, then there are a lot more kinds of such people to model, and I don't think we are ready for this yet.

Reply:
Originally when I put in the owner field, I was thinking of it for RBAC purposes. In that case owner does not necessarily need to be a single individual owner, it could be a reference to kind of group in your AuthN system. However now I think this maybe more suited as an annotation and not as part of the spec.

Reply:
Yeah access control to entities of all kinds is an interesting topic of its own - not sure that those permissions will be encoded in the entities themselves usually! They can be parts of a separate RBAC system, or based on your membership type in the group (internally we tag people in a group as one or both of "owner" and "member", for example, where the former can modify the group).